app/models/resource.py

```diff
@@ -98,8 +99,28 @@ class Resource(db.Model):
         back_populates='resource',
         cascade='save-update, merge, delete, delete-orphan'
     )
-    suggestions = db.relationship('Suggestion', backref='resource',
-                                  uselist=True)
+    type = db.Column(db.String(20))
+
+    __mapper_args__ = {
+        'polymorphic_on': type,
+        'polymorphic_identity': 'resource_base'
```

Comment:
This doc says both are needed for the base class: http://docs.sqlalchemy.org/en/latest/orm/inheritance.html#single-table-inheritance